roles/reproducer/tasks/push_code.yml

```diff
@@ -123,6 +126,26 @@
         loop_var: repository
         label: "{{ repository.src | basename }}"
 
+- name: Push code in ~/src to the controller-0
+  when:
+    - cifmw_reproducer_skip_fetch_repositories | default(false)
+  block:
+    - name: Ensure repositories path exists in the controller-0
+      delegate_to: controller-0
+      ansible.builtin.file:
+        path: "/home/zuul/{{ cifmw_reproducer_repositories_path }}"
+        state: directory
+
+    - name: Copy code from ansible controller to controller-0
+      delegate_to: localhost
+      vars:
+        _source_path: "{{ cifmw_reproducer_repositories_path | regex_replace('/$', '') }}/"
```

Comment:
Shouldn't this var be used, with the / sanitized, be used in the `ansible.builtin.file` call?

Reply:
indeed, changed the block to use the var